testnet/stacks-node/src/config.rs

```diff
@@ -569,11 +571,105 @@ impl Config {
                     rbf_fee_increment: burnchain
                         .rbf_fee_increment
                         .unwrap_or(default_burnchain_config.rbf_fee_increment),
-                    epochs: match burnchain.epochs {
-                        Some(epochs) => Some(epochs),
-                        None => default_burnchain_config.epochs,
-                    },
+                    epochs: default_burnchain_config.epochs,
+                };
+
+                if let Some(ref conf_epochs) = burnchain.epochs {
```

Comment:
This code body is getting pretty long.  Maybe this block should be split out into its own method?

Reply:
Yep, good point, refactored in e852ab8